docs/design/storage.md

```diff
@@ -0,0 +1,99 @@
+# Storage
+
+KubeDiag supports storing the processing results of processors in minio.
+
+## Background
+
+Users can store the diagnosis result file in minio for later review by configuring the operation storage mode.
+
+## Implementation
+
+Enabling minio storage requires configuring the parameters `minio-endpoint`, `minio-access-key-id`, `minio-secret-access-key`, and `minio-ssl` in the agent.
```

Comment:
The high level abstraction of storage drivers should be emphasized in a design document.